File: Solutions/2886/c++/program.cpp
// Copy from Quera

#include <iostream>
#include <stdio.h>

using namespace std;

int main()
{
    int a, b;
    cin >> a >> b;
    printf("%02d:%02d\n", (12 - a) % 12, (60 - b) % 60);
    return 0;
}

File: Solutions/2886/java/program.java
// Copy from Quera

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        
        Scanner scanner=new Scanner(System.in);
        
        int hour = scanner.nextInt(), minute = scanner.nextInt();
        
        hour = 12 - hour;
        hour = hour % 12;
        
        minute = 60 - minute;
        minute = minute % 60;
        
        System.out.printf("%02d:%02d", hour, minute);
    }
}